linux-setup.sh

```diff
@@ -73,6 +73,13 @@ install_mkcert() {
 
         # cleanup temporary build directory
         rm -rf "$builddir"
+
+        mkcert -install
+
+        echo "You have installed mkcert (used to make khanacademy.dev work)"
+        echo "A CA has been added to your system and browser certificate trust stores."
+        echo ""
+        echo "You must REBOOT your machine for browsers to recognize new CA."
```

Comment:
https://kifarunix.com/create-locally-trusted-ssl-certificates-with-mkcert-on-ubuntu-20-04/ seems to indicate this isn't necessary.  But you may need to restart your browser.  I'd probably change this to "You may need to REBOOT ..."

Reply:
Thanks for the link.  I'll update the message.  MacOS must be different in this regard since rebooting is required on that platform.